opentelemetry-zipkin/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "opentelemetry-zipkin"
-version = "0.21.0"
+version = "0.22.0"
```

Comment:
version bump is done once before crate publishing, we can revert this change for now.